examples/language_id_local_media_file.js

```diff
@@ -0,0 +1,50 @@
+const revai = require('revai-node-sdk');
```

Comment:
I wonder if it's necessary in general to have 2 separate examples for local media file vs url submission. We could combine the two with sections commented out and leave it up to the users to comment in the parts they want. But, maybe this would make the file too large and confusing